parlai/core/torch_agent.py

```diff
@@ -533,14 +543,12 @@ def batchify(self, obs_batch, sort=False,
         xs, x_lens = None, None
         if any('text_vec' in ex for ex in exs):
             _xs = [ex.get('text_vec', self.EMPTY) for ex in exs]
-            xs, x_lens = padded_tensor(_xs)
+            xs, x_lens = padded_tensor(_xs, self.NULL_IDX, self.use_cuda)
```

Comment:
I'd really like to add a `left_padded=True` here, so that the `xs` are left padded instead of right padded. This is more "standard" in seq2seq architectures, but I think y'all report it not mattering much?

Reply:
great!